packages/react-instantsearch-core/src/core/createInstantSearchManager.js

```diff
@@ -3,6 +3,15 @@ import createWidgetsManager from './createWidgetsManager';
 import createStore from './createStore';
 import { HIGHLIGHT_TAGS } from './highlight';
 import { hasMultipleIndices } from './indexUtils';
+import { version as ReactVersion } from 'react';
+import version from './version';
+
+export function addAlgoliaAgents(searchClient) {
```

Comment:
Why do we export the function?

Reply:
This was because it was in utils first, good catch